vscode-dotnet-runtime-extension/src/extension.ts

```diff
@@ -1,516 +1,519 @@
+import * as vscode from 'vscode';
+import {
+    AcquireErrorConfiguration,
+    AcquisitionInvoker,
+    callWithErrorHandling,
+    DotnetAcquisitionMissingLinuxDependencies,
+    DotnetAcquisitionRequested,
+    DotnetAcquisitionStatusRequested,
+    DotnetCoreAcquisitionWorker,
+    DotnetCoreDependencyInstaller,
+    DotnetExistingPathResolutionCompleted,
+    enableExtensionTelemetry,
+    EventBasedError,
+    ErrorConfiguration,
+    ExistingPathResolver,
+    ExtensionConfigurationWorker,
+    formatIssueUrl,
+    IDotnetAcquireContext,
+    IAcquisitionWorkerContext,
+    NoExtensionIdProvided,
+    IDotnetAcquireResult,
+    IDotnetEnsureDependenciesContext,
+    IDotnetUninstallContext,
+    IEventStreamContext,
+    IExtensionContext,
+    IIssueContext,
+    InstallationValidator,
+    registerEventStream,
+    directoryProviderFactory,
+    VersionResolver,
+    VSCodeExtensionContext,
+    VSCodeEnvironment,
+    WindowDisplayWorker,
+    GlobalInstallerResolver,
+    CommandExecutor,
+    IDotnetListVersionsContext,
+    WebRequestWorker,
+    IDotnetVersion,
+    DotnetInstallMode,
+    DotnetVersionResolutionError,
+    IDotnetListVersionsResult,
+    LinuxVersionResolver,
+    GlobalAcquisitionContextMenuOpened,
+    UserManualInstallVersionChosen,
+    UserManualInstallRequested,
+    UserManualInstallSuccess,
+    UserManualInstallFailure,
+    DotnetInstall,
+    EventCancellationError,
+    getInstallKeyCustomArchitecture,
+    DotnetInstallType,
+    DotnetAcquisitionTotalSuccessEvent,
+    isRunningUnderWSL
+} from 'vscode-dotnet-runtime-library';
+import { dotnetCoreAcquisitionExtensionId } from './DotnetCoreAcquisitionId';
+
+// tslint:disable no-var-requires
+const packageJson = require('../package.json');
+
+// Extension constants
+namespace configKeys {
+    export const installTimeoutValue = 'installTimeoutValue';
+    export const enableTelemetry = 'enableTelemetry';
+    export const existingPath = 'existingDotnetPath';
+    export const existingSharedPath = 'sharedExistingDotnetPath'
+    export const proxyUrl = 'proxyUrl';
+}
+namespace commandKeys {
+    export const acquire = 'acquire';
+    export const acquireGlobalSDK = 'acquireGlobalSDK';
+    export const acquireStatus = 'acquireStatus';
+    export const uninstallAll = 'uninstallAll';
+    export const listVersions = 'listVersions';
+    export const recommendedVersion = 'recommendedVersion'
+    export const globalAcquireSDKPublic = 'acquireGlobalSDKPublic';
+    export const showAcquisitionLog = 'showAcquisitionLog';
+    export const ensureDotnetDependencies = 'ensureDotnetDependencies';
+    export const reportIssue = 'reportIssue';
+}
+
+const commandPrefix = 'dotnet';
+const configPrefix = 'dotnetAcquisitionExtension';
+const displayChannelName = '.NET Install Tool';
+const defaultTimeoutValue = 600;
+const moreInfoUrl = 'https://github.com/dotnet/vscode-dotnet-runtime/blob/main/Documentation/troubleshooting-runtime.md';

```

Comment:
Should this always link to `main` or some static commit? If you go to that URL right now and then hit `y`, it'll change your address bar to have the commit ID in the URL. It turns into `https://github.com/dotnet/vscode-dotnet-runtime/blob/e68687f05ea3f59dc08437bfaa6c4d07774cf953/Documentation/troubleshooting-runtime.md` for me right now. You should also use an aka.ms link (let me know if you need help creating them).

Reply:
I think it should default to `main` so it always provides the latest documentation if we ever improve the document.

The aka.ms link idea is a good one :) I will create another issue to do that. I appreciate that you offered to help, I'll see what I can do first! https://github.com/dotnet/vscode-dotnet-runtime/issues/1867

Reply:
The reason to not link to main is that file path changes (or renames) would make the URL no longer valid. But, if you use an aka.ms link, then main is fine as the forwarding link can be updated without issue if the file is moved/renamed.